cvmfs/receiver/payload_processor.cc

```diff
@@ -67,11 +67,15 @@ void PayloadProcessor::ConsumerEventCallback(
     path = event.object_name;
   } else {
     // kEmpty - this is an error.
-    LogCvmfs(kLogCvmfs, kLogStderr, "Event received with unknown object.");
+    LogCvmfs(kLogReceiver, kLogDebug | kLogSyslogErr,
+             "Event received with unknown object.");
     num_errors_++;
     return;
   }
 
+  LogCvmfs(kLogReceiver, kLogDebug | kLogSyslog,
```

Comment:
Probably the same for https://github.com/cvmfs/cvmfs/pull/1902/files#diff-1e4aca86201436e103fcd7ff1b7ff156R51